tools/setool/README.md

```diff
@@ -0,0 +1,31 @@
+# SETool
```

Comment:
Hey, why is this included in this PR?

Reply:
My bad, I merged previously this on my branch. This is actually reviewed in https://github.com/CartoDB/carto-spatial-extension/pull/73/files